src/crypto/crypto_util.cc

```diff
@@ -120,10 +120,9 @@ void InitCryptoOnce() {
     }
   }
   if (0 != err) {
-    fprintf(stderr,
-            "openssl fips failed: %s\n",
-            ERR_error_string(err, nullptr));
-    UNREACHABLE();
+    auto *isolate = Isolate::GetCurrent();
+    auto *env = Environment::GetCurrent(isolate);
+    return ThrowCryptoError(env, err, ERR_error_string(err, nullptr));
```

Comment:
This could optionally be shortened to just be:
```c++
return ThrowCryptoError(env, err); 
```

Reply:
I assume you mean the return line can be shortened… otherwise we are back at the start with not having an environment pointer – or do I miss something?

Reply:
Yeah, sorry that is what I meant, just the last line.